setup_wifi.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+
+# Script for changing MultiAP WiFi settings in Teltonika RUTX11.
+# Requires RUTX11 device with configuration set by first_setup.sh script - by default done by Husarion during production.  
+
+import io
+import json
+from pssh.clients import SSHClient
+import pssh.exceptions 
+import subprocess
+import time
+import os
+import sys
+
+allowed_radio = ['0', '1']
+
+host = '10.15.20.1' # IP adress of RUTX11
+user_name = "root"
+config_file = 'config.json' 
+reconfigure_wifi = False
+reconfigure_client = False
+debug_flag = False # if true print debug messages
+
+ssh = SSHClient(host, user=user_name, timeout=5, num_retries=1)
+
+def multi_wifi_config_validator(data, name):
+    index = 1
+    for x in data:
```

Comment:
```suggestion
    for index, key in enumerate(data, start=1):
```
This can be simplified. This way you don't have to worry about incrementing index later.